src/components/listview/listview.js

```diff
@@ -148,11 +149,7 @@ import ServerConnections from '../ServerConnections';
 
             elem.classList.add('listItemBodyText');
 
-            if (textlines[i]) {
-                elem.innerText = textlines[i];
-            } else {
-                elem.innerHTML = '&nbsp;';
-            }
+            elem.innerText = text;
```

Comment:
Should this be `text || '&nbsp;`?

Reply:
This is unnecessary because `text` is defined and non-empty:
https://github.com/jellyfin/jellyfin-web/blob/8420d0fef20851d259df891289bebaf5ea342a7c/src/components/listview/listview.js#L133-L135